spec/UseAtomSpec.php

```diff
@@ -14,13 +14,18 @@
         \WP_Mock::tearDown();
     });
 
-    it('should register actions correctly', function() {
-        // expect($this->UseAtom)->toBeAnInstanceOf(\Dxw\Iguana\Registerable::class);
-        \WP_Mock::expectActionAdded('init', [$this->useAtom, 'init']);
-        \WP_Mock::expectActionAdded('wp_head', [$this->useAtom, 'wpHead']);
+    describe("::register()", function() {
+        it('should register actions correctly', function() {
+            // expect($this->UseAtom)->toBeAnInstanceOf(\Dxw\Iguana\Registerable::class);
+            \WP_Mock::expectActionAdded('init', [$this->useAtom, 'init']);
```

Comment:
I'm not sure we need `WP_Mock` here for Kahlan tests, have a look at other repos where we use Kahlan and see if you can do the same thing with `allow()` which is more idiomatic in Kahlan.